wgpu-types/src/lib.rs

```diff
@@ -388,6 +388,13 @@ bitflags::bitflags! {
         ///
         /// This is a native-only feature.
         const TEXTURE_ADAPTER_SPECIFIC_FORMAT_FEATURES = 0x0000_0000_1000_0000;
+        /// Enables 64-bit floating point types in SPIR-V shaders.
```

Comment:
This really should carry a warning about the anemic throughout our doubles in shaders. If people are used to the float perf, they're going to get burned here.